source/numerics.tex

```diff
@@ -16034,3 +16034,2871 @@
                                      A, t, d, B, divides<void>{});
 \end{codeblock}
 \end{itemdescr}
+
+\rSec1[simd]{Data-parallel types}
+
+\rSec2[simd.general]{General}
+
+\pnum
+[simd] defines data-parallel types and operations on these types.
```

Comment:
`[simd]` isn't a number:
```suggestion
Subclause\iref{simd} defines data-parallel types and operations on these types.
```

Reply:
This is the exact wording as given in LWG review in St. Louis. But I believe the intent was to mirror https://eel.is/c++draft/linalg. So :+1:

Reply:
I hope nobody believes putting stable labels verbatim in the standard text is a good idea, ever.  For reference, [linalg.overview] says

"Subclause 29.9 defines basic linear algebra algorithms. ..."

I have no idea why eel.is has verbatim "[linalg]" in there.  @Eelis , looks like a bug.

Reply:
cxxdraft-htmlgen renders section references as abbreviations rather than numbers in order to make citations more informative/self-contained. For example,

> ... as specified in [expr.new].
    
is more informative/self-contained than

> ... as specified in 7.6.2.8.

Reply:
@Eelis, right.  But in this case, [linalg] is not a hyperlink, it's just dumb text. In other situations, the label-reference is a hyperlink (see later in that sentence, for example).